common/wuwebview/wuwebview.cpp

```diff
@@ -356,6 +358,7 @@ class WuWebView : public CInterface,
     Owned<IConstWorkUnit> cw;
     Owned<ILoadedDllEntry> dll;
     Owned<IPropertyTree> manifest;
+    Owned<IPropertyTree> xsltConfig;
```

Comment:
You could have used Linked<> instead (and then initialized via constructor initializer list). Might make the ownership a teeny bit clearer.

Reply:
Agreed, I'll change it.